environments/sandbox.yml

```
@@ -102,7 +102,8 @@ cname:
   - name: "plum"
     ttl: 300
     record: "hmcts-sbox.azurefd.net"
-    shutter: false
+    shutter: true
```

i don't think you want both of these true?
```suggestion
    shutter: false
```